turbinia/config/logger.py

```diff
@@ -62,7 +67,15 @@ def setup(need_file_handler=True, need_stream_handler=True):
               exception, config.CONFIG_MSG))
       sys.exit(1)
 
-    file_handler = logging.FileHandler(config.LOG_FILE)
+    # Check if user specified log name and update to default if not
+    if not user_specified_log:
+      log_name = uname().nodename
+      # Check if NODE_NAME available for GKE setups
+      if ENVNODENAME in environ:
+        log_name = log_name + '.{0!s}'.format(environ[ENVNODENAME])
+      config.LOG_DIR = '{0:s}/{1:s}.log'.format(config.LOG_DIR, log_name)
```

Comment:
Another very minor nit, but usually `os.path.join` is safer or more idiomatic than concatenating the strings with the path separator.

Reply:
Updated to os.path.join